### ISSUE TITLE
State contributions more clearly

### ISSUE THREAD

**Issue by gregmacfarlane** [gregmacfarlane/parks_access#9]
Reviewer 2:

> The first two sections (introduction and literature review) provide a nice overview of past research in this area and different approaches to modeling park accessibility. However, it is not clearly stated what is missing from this body of literature or how this study fills an important gap. Likewise, the manuscript's explicit purpose could/should be stated more explicitly (beyond the "we" statements in the introduction) and likely reiterated definitively after past literature is reviewed. Without these elements, the contribution(s) of the manuscript is/are murky.
